### PR TITLE
add pulltorefreshjs@0.1.0 w/ npm auto-update

### LINKED ISSUE #9898: [Request] Add pulltorefreshjs
**Library name:** pulltorefreshjs
**Git repository url:** https://github.com/BoxFactura/pulltorefresh.js
**npm package url(optional):** https://www.npmjs.com/package/pulltorefreshjs
**License(s):** MIT
**Official homepage:** https://www.boxfactura.com/pulltorefresh.js
**Wanna say something? Leave message here:** A small, but powerful Javascript library crafted to power your webapp's pull to refresh feature. No markup needed, highly customizable and dependency-free!

### PR DESCRIPTION
**Pull request by kennynaoh**
Pull request for issue: #9898
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.
@x09326 Please help me review this PR, thanks!
Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/BoxFactura/pulltorefresh.js
 * Official website (optional, not the repository): https://www.boxfactura.com/pulltorefresh.js/
 * NPM package url (optional): https://www.npmjs.com/package/pulltorefreshjs
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 870
   - Count of watchers: 18
   - Count of forks: 25
 * NPM download stats (optional):
   - Downloads in the last day: 16
   - Downloads in the last week: 16
   - Downloads in the last month: 16

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9898, cc @pateketrueke

### CODE REVIEW

**Comment by x09326** [comment]
@kennynaoh 
Please rebase to the latest master branch.

**Comment by kennynaoh** [comment]
@x09326 I have rebased to the latest master branch. Please help me review it again. Thanks!

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

File: ajax/libs/pulltorefreshjs/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "pulltorefreshjs",
+  "filename": "pulltorefresh.min.js",
+  "version": "0.1.0",
+  "description": "Pull To Refresh",
+  "keywords": [
+    "pulltorefresh",
+    "webapp",
+    "javascript"
```
Comment (PeterDaveHello) [inline ajax/libs/pulltorefreshjs/package.json]: @kennynaoh do you made the keywords yourself? I don't think "javascript" is good to be a keyword.

**Comment by kennynaoh** [comment]
@x09326 I have removed `javascript` in `keywords` field. Please help me review it again. Thanks!

**Pull request merged**